css/styles.css

```diff
@@ -72,9 +75,142 @@ ul {
 
 .icons {
   display: flex;
-  align-items: center;
 }
 
 .icons li {
   padding-right: 12px;
 }
+
+.workTitle {
+  font-family: Crete-Round, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  width: 375px;
+  left: 0;
+  top: 682px;
+}
+
+.works {
+  margin-left: 40px;
+  margin-right: 40px;
+}
+
+.yoga {
+  margin-top: 30px;
+}
+
+.border {
+  width: 327px;
+  height: 0;
+  left: 0;
+  top: 107px;
+  border: 1px solid #212121;
+}
+
+.secondPage {
+  background: #e5e5e5;
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.secondItem {
+  background-image: url('secondBG.png');
+  background-size: cover;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-top: 40px;
+  padding: 20px;
+}
+
```

Comment:
nice work so far! 😃 
your contents in the container are tilting to the left or the right of the page
The project description is also overflowing its container, which makes the page not appear presentable.
try and make sure every text does not overflow the width of its container.
you can also use `justify-content: center` to `margin: auto` to make sure your page is justified.
![image](https://user-images.githubusercontent.com/43564324/164755175-406ced4c-b62d-47be-9572-1e661a628910.png)

Reply:
Hello, I have a couple of questions.

- What is the size of the screen that you are using to view this page? 

- How can I make sure the content doesn't overflow the container?

- When you say

"you can also use justify-content: center to margin: auto to make sure your page is justified."
Do you mean to change 'justify-content: center' to 'margin: auto', or to set both to my page? And do I add this to the entire body of the page?

Thank you.

Reply:
you can use any depending if the container is `display: flex` or `display:grid` you can use `justify-content: center` to put the direct child in center but I would advise you to use justify-content so you improve your skills in using flex and grid

Reply:
if you justify the content and it's wrapped in a container with flexible width in percentage, it should not overlap  to the right or tilt in any screen size

Reply:
what do you mean by flexible width in percentage?